frontend/src/components/Sheet/Sheet.module.scss

```diff
@@ -25,6 +25,34 @@
   color: $secondary-yellow;
 }
 
+.containerDropdownNavbar {
+  z-index: 3;
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  width: 100vw;
```

Comment:
maybe make this width not cover the whole screen? Like maybe just on the right. In my 3900 project we made it slide from the right (like a drawer) which I liked but dw if u can't be bothered.